README.md

```diff
@@ -5,6 +5,11 @@ Alpakka: Akka Streams Connectors
 
 This project provides a home to Akka Streams connectors to various technologies, protocols or libraries.
 
+Documentation
+-------------
+
+Latest [documentation](http://developer.lightbend.com/docs/alpakka/latest/).
+
```

Comment:
@2m What is the link to published Scaladoc? Do we have Javadoc (can be solved later)?